doc/Command-Reference.md

```diff
@@ -4108,6 +4108,37 @@ This command is used to delete the syslog server configured.
   Restarting rsyslog-config service...
   admin@str-s6000-acs-11:~$
   ```
+
+# DHCP Relay Destination Server Configuration Commands 
```

Comment:
I think we should change "server" to "IP address", or remove it altogether, since the destination IPs are not necessarily the addresses of servers, but could potentially be "helper" IPs which point to a subnet, etc. This applies to nomenclature used throughout this PR.